packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -246,6 +255,10 @@ public void init(
         if (!Strings.isNullOrEmpty(hostedDomain)) {
           optionsBuilder.setHostedDomain(hostedDomain);
         }
+        if (!Strings.isNullOrEmpty(serverClientId)) {
+          optionsBuilder.requestServerAuthCode(serverClientId, true);
```

Comment:
If we need to expose this API, let's also expose the boolean parameter `forceCodeForRefreshToken` to the public API and make it `false` by default.